panpipes/python_scripts/batch_correct_mofa.py

```diff
@@ -95,6 +95,21 @@
 L.info("Intersecting modality obs before running mofa")
 mu.pp.intersect_obs(tmp)
 
+
+if check_for_bool(params["multimodal"]["MultiVI"]["lowmem"]):
```

Comment:
@wlason @bio-la  shouldnt this be` ['"multimodal''][''mofa'']["lowmem"]`

Reply:
yes! and that's exactly the issue that a CI would catch. let me fix this in the meantime.